src/pages/host/Reviews.jsx

```diff
@@ -0,0 +1,76 @@
+import chart from "../../assets/images/review-chart.png";
+const Reviews = () => {
+  const reviews = [
+    {
+      name: "Elliot",
+      date: "December,1,2022",
+      content:
+        "The beach bum is such as awesome van! Such as comfortable trip. We had it for 2 weeks and there was not a single issue. Super clean when we picked it up and the host is very comfortable and understanding. Highly recommend!",
+    },
+    {
+      name: "Sandy",
+      date: "November,23,2022",
+      content:
+        "This is our third time using the Modest Explorer for our travels and we love it! No complaints, absolutely perfect!",
+    },
+  ];
+  return (
+    <div className="space-y-6">
+      <div className="flex flex-wrap items-center justify-between gap-4">
+        <p className="text-2xl font-bold">Your Reviews</p>
+        <p className="font-normal text-gray-600">
+          last<span className="px-1 font-bold underline">30 days</span>
+        </p>
+      </div>
+      <p className="text-2xl font-bold">
+        5.0
+        <span className="px-1 text-base font-normal text-gray-600 underline">
+          overall rating
+        </span>
+      </p>
+      <img
+        src={chart}
+        alt="review chart"
+        className="object-cover object-center max-h-44 "
+      />
+      <p className="text-2xl font-bold">Reviews(2)</p>
+      <div className="space-y-4">
+        {reviews.map((review, index) => (
+          <div
+            key={index}
+            className="max-w-2xl p-4 space-y-2 border-b-2 border-gray-400"
+          >
+            <div className="flex flex-wrap gap-1">
+              {[1, 2, 3, 4, 5].map((star, index) => (
```

Comment:
I suggest you can use this
```suggestion
              {Array(5).fill().map((star, index) => (
              
```